### PR TITLE
ITEM 20150717_005556 - 005569

### PR DESCRIPTION
**Pull request by WorksWellWithOthers**
ITEM 20150717_005556 - 005569 translated.

### CODE REVIEW

File: ITEM.tsv [hunk 1]
```diff
@@ -5553,20 +5553,20 @@ ITEM_20150717_005552	There is not much like this to have as your guard.
...
-ITEM_20150717_005562	
-ITEM_20150717_005563	
-ITEM_20150717_005564	
-ITEM_20150717_005565	
-ITEM_20150717_005566	
-ITEM_20150717_005567	
-ITEM_20150717_005568	
-ITEM_20150717_005569	
+ITEM_20150717_005560    We have found death without discretion. It is time to apply what we have discovered on the enemy.	
+ITEM_20150717_005561    You can't always draw power from wandering souls. However, if you succeed. An item that which increases your hidden potential will be created.
```
Comment (laurencee) [inline ITEM.tsv]: The last 2 sentences should be merged and duplicate word choice removed:
"However, if you succeed an item which increases your hidden potential will be created."

File: ITEM.tsv [hunk 2]
```diff
@@ -5553,20 +5553,20 @@ ITEM_20150717_005552	There is not much like this to have as your guard.
...
+ITEM_20150717_005562    The craftsman takes pride in bringing out the optimum curvature of this recurve bow.
+ITEM_20150717_005563    The Star Tower archers do not care for this bow. However, others living in these treacherous times will find this bow useful. 	
+ITEM_20150717_005564    Just because it is called the sniper's bow doesn't mean you have to take single shots.	
+ITEM_20150717_005565    During perilous times, if secret and terror are wielded in the shooter's hands, they will draw out the will to live.	
+ITEM_20150717_005566    The first time a device was attached to the tip of a bow to increase it's draw power was during Lydia Schaffen's era. The actual creator of the device was a master fletcher, who was also Lydia Schaffen's ally.	
+ITEM_20150717_005567    This bow requires a lot more strength from the user than expected. The view of the arrow in flight will leave a sense of satisfaction.	
+ITEM_20150717_005568    Unlike its name, it will be welcomed by any archer. Only rogues dislike it.	
+ITEM_20150717_005569    The burden of both hands and bow have been alleviated, it does not decrease the power by much.
```
Comment (laurencee) [inline ITEM.tsv]: This line does not sound right... is it meant to be "The burden of 2 handed bows has been alleviated"?

**Comment by WorksWellWithOthers** [comment]
It should be "The burden of two handed bows has been alleviated". Except that all bows are two handed.

**Comment by laurencee** [comment]
:+1:

File: ITEM.tsv [hunk 3]
```diff
@@ -5553,20 +5553,20 @@ ITEM_20150717_005552	There is not much like this to have as your guard.
...
+ITEM_20150717_005562    The craftsman takes pride in bringing out the optimum curvature of this recurve bow.
+ITEM_20150717_005563    The Star Tower archers do not care for this bow. However, others living in these treacherous times will find this bow useful. 	
+ITEM_20150717_005564    Just because it is called the sniper's bow doesn't mean you have to take single shots.	
+ITEM_20150717_005565    During perilous times, if secret and terror are wielded in the shooter's hands, they will draw out the will to live.	
+ITEM_20150717_005566    The first time a device was attached to the tip of a bow to increase it's draw power was during Lydia Schaffen's era. The actual creator of the device was a master fletcher, who was also Lydia Schaffen's ally.	
+ITEM_20150717_005567    This bow requires a lot more strength from the user than expected. The view of the arrow in flight will leave a sense of satisfaction.	
+ITEM_20150717_005568    Unlike its name, it will be welcomed by any archer. Only rogues dislike it.	
+ITEM_20150717_005569    The burden of two handed bows has been alleviated, it does not decrease the power by much.
```
Comment (Soukyuu) [inline ITEM.tsv]: @"all bows are two handed": I think this is to differentiate between 2h bows (=bows) and 1h bows (=crossbows). There are a couple of other places where this formulation is used, but I think it's safe to replace "two handed bows" by "bows".

**Comment by Soukyuu** [comment]
:+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**